templates/certified/model-details/_components-section.html

```diff
@@ -1,113 +1,84 @@
+        <colgroup>
+          <col class="col-component" />
+          <col class="col-release" />
+          <col class="col-status" />
+          <col class="col-comments" />
+        </colgroup>
+
+        <thead>
+          <tr>
+            <th scope="col">Component</th>
+            <th scope="col">Ubuntu release</th>
+            <th scope="col">Status</th>
+            <th scope="col">Comments</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {% set releases = ["24.04 LTS", "22.04 LTS", "20.04 LTS", "CORE 20", "18.04 LTS", "CORE 18"] %}
+          {% for component in components %}
+            {% for i in range(releases|length) %}
+              {% set release = releases[i] %}
+              <tr>
+                {% if i == 0 %}
+                  <th scope="rowgroup" rowspan="{{ releases|length }}">
+                    <a href="/certified/component/{{ component.id }}">{{ component.vendor_name }} {{ component.model }}</a>
+                  </th>
+                {% endif %}
+                <td>{{ release }}</td>
+                {% if release in component.lts_releases %}
+                  {% set rel_data = component.lts_releases[release][0] %}
+                  {% if rel_data.status == 'inprogress' %}
+                    <td data-heading="{{ release.split(" ")[0] }} ESM" class="p-inline">
+                      <i class="p-icon--help">In progress</i>
+                      <span>In progress</span>
+                      {% if rel_data.third_party_driver %}<span>- may require 3rd-party driver</span>{% endif %}
+                    </td>
+                  {% elif rel_data.status == 'certified' %}
+                    <td data-heading="{{ release }}" class="p-inline">
+                      <i class="p-icon--success">Supported</i>
+                      <span>Supported</span>
+                      {% if rel_data.third_party_driver %}<span>- may require 3rd-party driver</span>{% endif %}
+                    </td>
+                  {% elif rel_data.status == 'unsupported' %}
+                    <td data-heading="{{ release }}" class="p-inline">
+                      <i class="p-icon--error"></i>
+                      <span>Unsupported</span>
+                      {% if rel_data.third_party_driver %}<span>- may require 3rd-party driver</span>{% endif %}
+                    </td>
+                  {% endif %}
+                {% else %}
+                  <td data-heading="{{ release }}">–</td>
+                {% endif %}
+                {% if i == 0 %}
+                  <td data-heading="Comments"
+                      scope="rowgroup"
+                      rowspan="{{ releases|length }}">{{ component.note }}</td>
+                {% endif %}
+              </tr>
+            {% endfor %}
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+    <div>
+      <hr class="p-rule" />
+      <p class="u-text--muted">*LTS releases with a blank status have not been tested</p>
```

Comment:
```suggestion
      <p class="u-text--muted">*Releases with a blank status have not been tested</p>
```
This applies to both LTS and Core releases